app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<nav class="first">
```

Comment:
- Kindly follow the design guidelines for the Login page:
    - [ ] Please add the `login link` in the navbar in the header responsible for the login process instead of adding a login button.

| Your design | Guidelines |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/230424788-5f2b99f3-d203-45ca-9700-64d84a63be0c.png) | ![image](https://user-images.githubusercontent.com/24703171/219885103-63122090-4832-4d4f-b31f-0fc2e80b3777.png) |